feat: add presentations page (#357)
open-feature/community: presentations.md

# Presentations

> This page showcases presentation given by the OpenFeature community, providing links to past presentations, and offering resources for community members who wish to contribute to future presentations.

Reviewer: ```suggestion
This page showcases presentations and interviews given by the OpenFeature community, providing links, and offering resources for community members who wish to contribute to future presentations.
```

## Previous presentations

> This section provides an overview of all the presentations given by the OpenFeature community, including links to recordings and slides where available.

Reviewer: ```suggestion
This section provides an overview of presentations and interviews given by OpenFeature community members, including links to recordings and slides where available.
```
